technicalreport/stateoftheart.tex

```diff
@@ -142,6 +142,20 @@ \subsubsection{Start-/Endpunkt auf der Fläche}
 
 Wie in Abbildung x zu sehen ergibt der zweite Ansatz ohne zusätzlichen Berechnungen genügend gute Resultate, da es zwischen jedem Ein- und Ausgangspunkt ein Fussweg hat, der bereits den kürzesten Weg beschreibt. Eine zusätzliche Berechnung des kürzesten Pfades wie im ersten Ansatz ergibt keine signifikante Verbesserung der Route.
 
+\subsubsection{Routing bei zwei benachbarten Flächen}
```

Comment:
wäre noch ein Bild hilfreich wo man so ein Beispiel sieht?

Reply:
In der referenzierten Ziel-Beschreibung haben wir schon ein Bild dazu